spec/models/post_spec.rb

```diff
@@ -1,5 +1,72 @@
-require 'rails_helper'
+require_relative '../rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:user) { User.create(name: 'Fuad', posts_counter: 0) }
+  subject do
+    Post.new(user_id: user.id, title: 'Hello', text: 'This is my post', comment_counter: 1, likes_counter: 1)
+  end
+
+  # before { subject.save }
```

Comment:
- [ ] Kindly remove all commented pieces of code from your documents. This is to ensure a clean and professional document.

Reply:
Hello @Kamba56, thanks for the review. The project has been updated to meet the requirement as stated above.